scala/defs.bzl

```diff
@@ -0,0 +1,40 @@
+"""
+Starlark rules for building Scala projects.
+
+These are the core rules under active development. Their APIs are
+not guaranteed stable and we anticipate some breaking changes.
+
+We do not recommend using these APIs for production codebases. Instead,
+use the stable rules exported by scala.bzl:
+
+```
+load(
+    "@io_bazel_rules_scala//scala:scala.bzl",
+    "scala_library",
+    "scala_binary",
+    "scala_test"
+)
+```
+
+"""
+
+load(
+    "@io_bazel_rules_scala//scala/private:rules/scala_binary.bzl",
+    _make_scala_binary = "make_scala_binary",
+)
+load(
+    "@io_bazel_rules_scala//scala/private:rules/scala_library.bzl",
+    _make_scala_library = "make_scala_library",
+)
+load(
+    "@io_bazel_rules_scala//scala/private:rules/scala_test.bzl",
+    _make_scala_test = "make_scala_test",
+)
+
+make_scala_library = _make_scala_library
```

Comment:
I believe these 6 APIs should be exported.

Right now it's the same exact thing exported by `scala.bzl` + the advance usage helpers, but they will likely deviate over the next year.